README.md

```diff
@@ -53,6 +53,12 @@ Reach project contributors via these channels:
  * [Gitter chat room](https://gitter.im/jaegertracing/Lobby)
  * [Github issues](https://github.com/uber/jaeger/issues)
 
+## Adopters
+
+Jaeger is [used in production](./ADOPTERS.md) by organizations such as Atlassian, Orange, Red Hat and Uber.
```

Comment:
@caniszczyk doesn't this statement conflate "adopting" with production usage? For example, vendors typically are not end users, though sometimes they are. For example, I'd be surprised if @postwait and others at circonus replaced their core infrastructure with jaeger, even if they have a zipkin-jaeger plug-in for one of their projects.

Similarly Red Hat do run production infrastructure I think, does this mean the public openshift infrastructure is monitored with jaeger? or does it mean they support and build plugins and tooling for others?

TL;DR; is I think this is "leading the witness" in so far as it is conflating a list of folks participating in the ecosystem with being production users. Users have skin in the game, when something breaks, their alerts go off. Production usually means things like this. Maybe it is better to just call it an adopters list and leave bullets in the list for people to offer information about whether they are in production or not?

I CNCF projects would have higher credibility if folks get accurate info about their projects.